.github/workflows/ci.yml

```diff
@@ -14,13 +14,17 @@ jobs:
     name: Tests
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        go-version: [1.14.x, 1.21.x]
```

Comment:
What happens if you run the jobs with version `1.21.x` only? Whats stopping us from only using that version?

Reply:
I'm not sure which services have all been upgraded yet, so I didn't want to cause any conflicts. I'm sure once we know all service that import this module are at or above 1.21 we can just update the module to that version.